projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/common/DataJobDeploymentExtension.java

```diff
@@ -124,7 +126,8 @@ public void beforeEach(ExtensionContext context) throws Exception {
 
     if (!initialized) {
       byte[] jobZipBinary =
-          IOUtils.toByteArray(getClass().getClassLoader().getResourceAsStream(jobSource));
+          IOUtils.toByteArray(
```

Comment:
Those zip files are a bit hard to review. I've been thinking it might be good idea to keep them as a directory and zip them here.

This is out of scope for this change and it would be better done in separate PR anyhow, just wondering what do you think.